src/config.py

```diff
@@ -7,83 +7,102 @@
+    optimizer_enable_async_append: bool | None = Field(default=None)
+    optimizer_enable_bitmapscan: bool | None = Field(default=None)
+    optimizer_enable_gathermerge: bool | None = Field(default=None)
+    optimizer_enable_hashagg: bool | None = Field(default=None)
+    optimizer_enable_hashjoin: bool | None = Field(default=None)
+    optimizer_enable_incremental_sort: bool | None = Field(default=None)
+    optimizer_enable_indexonlyscan: bool | None = Field(default=None)
+    optimizer_enable_indexscan: bool | None = Field(default=None)
+    optimizer_enable_material: bool | None = Field(default=None)
+    optimizer_enable_memoize: bool | None = Field(default=None)
+    optimizer_enable_mergejoin: bool | None = Field(default=None)
+    optimizer_enable_nestloop: bool | None = Field(default=None)
+    optimizer_enable_parallel_append: bool | None = Field(default=None)
+    optimizer_enable_parallel_hash: bool | None = Field(default=None)
+    optimizer_enable_partition_pruning: bool | None = Field(default=None)
+    optimizer_enable_partitionwise_aggregate: bool | None = Field(default=None)
+    optimizer_enable_partitionwise_join: bool | None = Field(default=None)
+    optimizer_enable_seqscan: bool | None = Field(default=None)
+    optimizer_enable_sort: bool | None = Field(default=None)
+    optimizer_enable_tidscan: bool | None = Field(default=None)
+    optimizer_from_collapse_limit: int | None = Field(ge=1, le=2147483647, default=None)
+    optimizer_geqo: bool | None = Field(default=None)
+    optimizer_geqo_effort: int | None = Field(ge=1, le=10, default=None)
+    optimizer_geqo_generations: int | None = Field(ge=0, le=2147483647, default=None)
+    optimizer_geqo_pool_size: int | None = Field(ge=0, le=2147483647, default=None)
+    optimizer_geqo_seed: float | None = Field(ge=0, le=1, default=None)
+    optimizer_geqo_selection_bias: float | None = Field(ge=1.5, le=2, default=None)
+    optimizer_geqo_threshold: int | None = Field(ge=2, le=2147483647, default=None)
+    optimizer_jit: bool | None = Field(default=None)
+    optimizer_jit_above_cost: float | None = Field(ge=-1, le=1.80e308, default=None)
+    optimizer_jit_inline_above_cost: float | None = Field(ge=-1, le=1.80e308, default=None)
+    optimizer_jit_optimize_above_cost: float | None = Field(ge=-1, le=1.80e308, default=None)
+    optimizer_join_collapse_limit: int | None = Field(ge=1, le=2147483647, default=None)
+    optimizer_min_parallel_index_scan_size: int | None = Field(ge=0, le=715827882, default=None)
+    optimizer_min_parallel_table_scan_size: int | None = Field(ge=0, le=715827882, default=None)
+    optimizer_parallel_setup_cost: float | None = Field(ge=0, le=1.80e308, default=None)
+    optimizer_parallel_tuple_cost: float | None = Field(ge=0, le=1.80e308, default=None)
+    profile: Literal["testing", "production"]
+    profile_limit_memory: int | None = Field(ge=128, le=9999999, default=None)
```

Comment:
Shouldn't profile be below plugin* options (alphabetically)?
LGTM, to merge AS IS, good to sort in the next PR.